python/tk_houdini/bootstrap.py

```diff
@@ -65,11 +65,12 @@ def bootstrap_classic():
         bootstrap_exception("Failed to import 'sgtk'!")
         return
 
-    # get the necessary environment variable for launch
-    env = get_classic_startup_env()
+    if g_temp_env not in os.environ:
+        # get the necessary environment variable for launch
+        env = get_classic_startup_env()
```

Comment:
This method creates the "TK_HOUDINI_TMP" env variable and appends it to the "HOUDINI_PATH" along with the classic startup directory path.

```
"HOUDINI_PATH": "C:/PROGRA~1/SIDEEF~1/HOUDIN~1.450/packages/kinefx;C:/Users/shotgrid/AppData/Local/Temp/tk-houdiniz_9z7uq6;C:/Users/shotgrid/repos/github.com/tk-houdini/classic_startup;&",
"TK_HOUDINI_TMP": "C:\\Users\\shotgrid\\AppData\\Local\\Temp\\tk-houdiniz_9z7uq6",
```